src/occ/parse/regs.h

```diff
@@ -43,8 +43,7 @@ int regmap[REG_MAX][2] = {
     {0, -1}, {1, -1}, {2, -1}, {3, -1}, {4, -1}, {5, -1}, {6, -1},  {7, -1},
 
 };
-static ARCH_REGDESC regNames[] = {
-    {"EAX", 0, 0, -1, 5, {8, 9, 16, 20, 24}},
+static ARCH_REGDESC regNames[] = {{"EAX", 0, 0, -1, 5, {8, 9, 16, 20, 24}},
```

Comment:
Please use original format/line break in this file. Thanks.

Reply:
I ran this through clang-format 7 and that's what seems to have messed EVERYTHING up, I have no idea why, but it seems to do so for some reason.

Sometimes I wonder if the dev tools I use are out to get me.